js/analytics.md

```diff
@@ -30,6 +30,8 @@ $ amplify push
 
 A configuration file called `aws-exports.js` will be copied to your configured source directory, for example `./src`. The CLI will also print the URL for Amazon Pinpoint console to track your app events.  
 
+**NOTE**: If your Analytics resources were created with Amplify CLI version below 1.6.5 (published on April 25, 2019), you will need to manually update your project to avoid Node.js runtime issues with AWS Lambda. [Read more]({%if jekyll.environment == 'production'%}{{site.amplify.docs_baseurl}}{%endif%}/cli/lambda-node-version-update)
```

Comment:
If your Analytics resources were created with Amplify CLI version 1.6.5 and below ? --> can you confirm

Reply:
1.6.5 is good, only versions below 1.6.5 needs update. 
yes I can confirm.

Reply:
I changed it to Amplify CLI version 1.6.4 and below